css/style.css

```diff
@@ -1,19 +1,203 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
 }
 
+body,
+button {
+  font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
+  color: var(--grey-dark);
+  font-size: 1rem;
+}
 /**
  * Add your custom styles below
- * 
+ 
  * Remember: 
  * - Be organised, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+h1 {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.header,
+.content,
+.footer {
+  padding: 0 1.5rem;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+/*Header*/
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 5rem;
+  margin-bottom: 0.2rem;
+  padding-top: 0.1rem;
+}
+
+.header__logo {
+  max-height: 70px;
+  width: 1.5rem;
+  padding-left: 100px;
+}
+
+/*Navigation*/
+
+nav {
+  margin-right: 10%;
+}
+.nav-list {
+  display: flex;
+  gap: 2rem;
+  list-style: none;
+  align-items: center;
+}
+
+.nav__item {
+  padding: 0.5rem 0;
+}
+
+.nav__link {
+  color: gray;
+  font-weight: 600;
+  text-transform: capitalize;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: orange;
+}
+
+/*Button*/
+
+button {
+  padding: 1rem 1rem 1rem 1rem;
```

Comment:
The concept of DRY code applies to the padding declaration as well. Instead of repeating "1rem" four times, you can simplify it by using the shorthand notation: `padding: 1rem;`. This approach achieves the same result and improves code readability.